### PR TITLE
chore: Fix docs/Simplify setup

### PR DESCRIPTION
**Pull request by lawrenceadams**
More can be done, but fix the mistakes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,17 +97,17 @@ dbt test
...
     dev:
       dbname: <name of local postgres DB>
       host: <host, e.g. localhost>
-      pass: <password>
+      user: <username>
+      password: <password>
       port: 5432
       schema: dbt_synthea_dev
-      threads: 1
+      threads: 8
```
Comment (lawrenceadams) [inline README.md:109]: Postgres can handle more than 1 connection.

File: README.md [hunk 2]
```diff
@@ -42,58 +42,52 @@ pre-commit install
...
 ```
 
- 8. Seed the location mapper and currently unused empty OMOP tables:
+ 7. Seed the location mapper and currently unused empty OMOP tables:
 ```bash
 dbt seed --select states omop
 ```
 
- 9. Build the OMOP tables:
+ 8. Build the OMOP tables:
```
Comment (katy-sadowski) [inline README.md:87]: make the same change for PG too?

File: README.md (hunk 1 above)
Comment (katy-sadowski) [reply README.md:109]: maybe we let users choose? and specify above they can choose appropriate # threads?

File: README.md [hunk 3]
```diff
@@ -42,58 +42,52 @@ pre-commit install
    - This will install dbt-core, the dbt duckdb and postgres adapters, SQLFluff (a SQL linter),  pre-commit (in order to run SQLFluff on all newly-committed code in this repo), duckdb (to support bootstrapping scripts), and various dependencies for the listed packages
 
 ### DuckDB Setup
- 1. Create a duckdb database in this repo's `data` directory (e.g. `data/synthea_omop_etl.duckdb`)
```
Comment (katy-sadowski) [inline README.md:45]: I assume this means that dbt will create the duckdb for you if it doesn't exist already?

**Review by katy-sadowski — COMMENTED** [review]
Thank you for this!  A couple of questions/comments.

File: README.md (hunk 1 above)
Comment (lawrenceadams) [reply README.md:109]: Good idea! For postgres 1 is a slow default! Let me add a note

File: README.md (hunk 3 above)
Comment (lawrenceadams) [reply README.md:45]: Yes exactly - if you run `dbt seed` before `dbt build/run` then it works fine!

To be honest - this is a slight point of taste/non-point, but I don't see the need to create a subdirectory to run it out of! (Bikeshedding territory - maybe just easier to leave as is)

Comment (katy-sadowski) [reply README.md:45]: Great! I'd used the data directory as a catchall so we could ignore any user-specific data in .gitignore. But putting the duckdb at root and ignoring *.duckdb works too.

File: README.md (hunk 2 above)
Comment (katy-sadowski) [reply README.md:87]: ready to approve this if you want to also make the change to `dbt build` in the postgres setup steps too, thanks!

**Review by katy-sadowski — APPROVED** [review]

**Pull request merged**